lib/scenic/statements.rb

```diff
@@ -21,7 +21,7 @@ module Statements
     #     SELECT * FROM users WHERE users.active = 't'
     #   SQL
     #
-    def create_view(name, version: nil, sql_definition: nil, materialized: false)
+    def create_view(name, version: nil, sql_definition: nil, materialized: false, materialized_no_data: false)
```

Comment:
That's a reasonable interface, especially given the current API for migrations. I agree about `materialized_no_data` => `no_data` changes in general.